baseapp/server.go

```diff
@@ -40,6 +45,8 @@ type Server struct {
 	// functions that are called once on start
 	initFns []func(*Server)
 	init    sync.Once
+
+	quit chan error
```

Comment:
It could be there as well. Initially I thought this might be useful if something else wanted to send a quit, but given that there isn't a use for this right now it should be moved.